cmd/dump.go

```diff
@@ -436,7 +439,7 @@ func extractOASApis(apis []objects.DBApiDefinition) (classic, oas []objects.DBAp
 	oas = []objects.DBApiDefinition{}
 
 	for i := 0; i < len(apis); i++ {
-		if apis[i].IsOASAPI() {
+		if apis[i].APIDefinition.IsOAS {
```

Comment:
I believe, here we might have nil pointer reference. it might be better to first check if `apis[i].APIDefinition` is not nil, then IsOAS. What do you think?
```suggestion
		if apis[i].APIDefinition != nil && apis[i].APIDefinition.IsOAS {
```

Reply:
I thought the same but forgot to do it.